### PR TITLE
fix i2c utc for read/write test

### PR DESCRIPTION
**Pull request by sangwon03**
I used the gy30 sensor for i2c test.
gy30 senser adress is 0x23 and it use just 1 byte packet for communication.
So I modified the adress and pakcet size.
And also I modified checking logic the result of read/write function because the return type of read/write is size.

### CODE REVIEW

File: .cproject [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (JeongJunSik) [inline .cproject]: Why do you upload this file?

File: .project [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (JeongJunSik) [inline .project]: Why do you upload this file?

Comment (JeongJunSik) [reply .project]: you need to delete this file also

File: apps/examples/testcase/ta_tc/systemio/utc/utc_i2c.c [hunk 3]
```diff
@@ -72,7 +72,7 @@ static void utc_i2c_set_frequency_n(void)
 
 static void utc_i2c_set_address_p(void)
 {
-	TC_ASSERT_EQ("iotbus_i2c_set_address", iotbus_i2c_set_address(i2c, 0x08), IOTBUS_ERROR_NONE);
+	TC_ASSERT_EQ("iotbus_i2c_set_address", iotbus_i2c_set_address(i2c, 0x23), IOTBUS_ERROR_NONE);  //for gy30 sensor
```
Comment (juitem) [inline apps/examples/testcase/ta_tc/systemio/utc/utc_i2c.c]: move description. with coding rule

File: apps/examples/testcase/ta_tc/systemio/utc/utc_i2c.c [hunk 4]
```diff
@@ -72,8 +72,7 @@ static void utc_i2c_set_frequency_n(void)
 
 static void utc_i2c_set_address_p(void)
 {
-	TC_ASSERT_EQ("iotbus_i2c_set_address", iotbus_i2c_set_address(i2c, 0x23), IOTBUS_ERROR_NONE);  
-	TC_SUCCESS_RESULT();
+	TC_ASSERT_EQ("iotbus_i2c_set_address", iotbus_i2c_set_address(i2c, 0x23), IOTBUS_ERROR_NONE);	TC_SUCCESS_RESULT();
```
Comment (JeongJunSik) [inline apps/examples/testcase/ta_tc/systemio/utc/utc_i2c.c]: you need to enter.

**Comment by juitem** [comment]
Even this commit is merged, 0x23 depends on specific h/w, this also should be changed on overall TCs

**Pull request merged**